### PR TITLE
New Feature: Events Manager. Some shortcodes were not included due to it...

### PR DESCRIPTION
**Pull request by oherman**
... being ineffective in page design, or is otherwise redundant.

### CODE REVIEW

File: lib/shortcode/events-manager.php [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+	if ( ! is_admin() ) {
+		return;
+	}
+	
+	if ( ! class_exists( 'EM_Scripts_and_Styles' ) ) {
+		return;
+	} 
+	
+	add_editor_style( EM_DIR_URI . 'includes/css/events_manager.css' );
```
Comment (bfintal) [inline lib/shortcode/events-manager.php]: Add check for `defined( 'EM_DIR_URI' )`

File: lib/shortcode/events-manager.php [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+					'value' => '10',
+					'description' => __( 'Show up to the specified amount of events.', 'pbsandwich' ),
+				),
+			),
+		)
+	);
+
+	// Register Shortcake UI for Events Manager Events List
+	shortcode_ui_register_for_shortcode(
+		'events_list',
```
Comment (bfintal) [inline lib/shortcode/events-manager.php]: There are 2 declarations for the shortcode `events_list`. Is the earlier one needed?

File: lib/shortcode/events-manager.php [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+	// Dependency check for Events Manager. Do not proceed if not found.
+	if ( ! defined( 'EM_VERSION' ) ) {
+		return;
+	}
+
+	// Register Shortcake UI for Events Manager Events List
+	shortcode_ui_register_for_shortcode(
+		'events_list',
+		array(
+			'label' => __( 'Events Manager - Events List', 'pbsandwich' ),
```
Comment (bfintal) [inline lib/shortcode/events-manager.php]: Remove the dash in the name of the shortcode since all our other third-party shortcodes do not have any dashes

**Comment by bfintal** [comment]
@oherman after you do all the changes please comment about and let @Gambit-Alyssa know when it's ready for testing

**Comment by oherman** [comment]
Changes done. Dupes spun off into a new function and requested changes are done.

@Gambit-Alyssa get ready to rip this apart.

**Comment by Gambit-Alyssa** [comment]
Events Manager: Events Submission Form
It shows differently in backend and frontend.

Backend View: http://prntscr.com/6in6el
Frontend View:
http://prntscr.com/6in7a5
http://prntscr.com/6in7eq

**Comment by Gambit-Alyssa** [comment]
Element's Text can be highlighted and moved around, moving the text makes the whole element go "poof".

Screencast: http://screencast-o-matic.com/watch/coeqXfesul

**Comment by oherman** [comment]
Different configs yield different styling. This might be a theme-dependent issue. Please try out with various theme and see if the same results occur.

**Comment by oherman** [comment]
Addressed by commit ec02d32

**Pull request merged**